cmd/geth/chaincmd.go

```diff
@@ -240,10 +240,9 @@ func initGenesis(ctx *cli.Context) error {
 		utils.Fatalf("invalid genesis file: %v", err)
 	}
 
-	// Open an initialise both full and light databases
 	stack, _ := makeConfigNode(ctx)
 	defer stack.Close()
-
+	// Open an initialise both full and light databases
```

Comment:
Code style nit, pls leave an empty line before a comment. Loosens the code up a bit :)

Reply:
"an" -> "and"

Reply:
Also I don't see why this comment was moved really. I thin the original place is correct. The role of the comment was to explain in general what the next section does.

Reply:
Mind you, you don't need an empty line there because `}` on itself is already loose enough :)

Reply:
Ah yeah, sorry for shifting these around, I was originally commenting as to why I was making a `makeConfigNode` rather than `makeFullNode` but we shifted those comments to the functions themselves. This was just legacy stuff. Just moved it back, thanks for the feedback!